src/items.rs

```diff
@@ -47,7 +47,9 @@ impl Rewrite for ast::Local {
     fn rewrite(&self, context: &RewriteContext<'_>, shape: Shape) -> Option<String> {
         debug!(
             "Local::rewrite {:?} {} {:?}",
-            self, shape.width, shape.indent
+            self,
+            shape.width,
+            shape.indent
```

Comment:
The change in `macros.rs` is apparently also resulting in these args being reformatted this way (the self_tests were failing with the original value). Not sure if this was expected and/or if it could be problematic

Reply:
Thank you for the catch. The change is undesirable, I think we could fix it by adding the following line to `OverflowableItem::is_simple`. Would you mind trying it out and update the PR?

```diff
    pub(crate) fn is_simple(&self) -> bool {
        match self {
            OverflowableItem::Expr(expr) => is_simple_expr(expr),
+           OverflowableItem::MacroArg(MacroArg::Keyword(..)) => true,
            OverflowableItem::MacroArg(MacroArg::Expr(expr)) => is_simple_expr(expr),
            OverflowableItem::NestedMetaItem(nested_meta_item) => match nested_meta_item {
                ast::NestedMetaItem::Literal(..) => true,
                ast::NestedMetaItem::MetaItem(ref meta_item) => match meta_item.node {
                    ast::MetaItemKind::Word => true,
                    _ => false,
                },
            },
            _ => false,
        }
    }
```

Reply:
Yup that worked 👍